src/components/otp-input/otp-input.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useRef } from 'react';
+import { HStack, Input } from 'native-base';
+
+interface OTPInputProps {
+  length: number;
+  otp: string[];
+  setOtp: (otp: string[]) => void;
+}
+
+const OTPInput: React.FC<OTPInputProps> = ({ length, otp, setOtp }) => {
+  const inputsRef = useRef<Array<any>>([]);
+
+  const handleChange = (text: string, index: number) => {
+    const newOtp = [...otp];
+    newOtp[index] = text;
+    setOtp(newOtp);
+
+    if (text && index < length - 1) {
+      inputsRef.current[index + 1].focus();
+    }
+  };
+
+  const handleKeyPress = (e: any, index: number) => {
+    if (e.nativeEvent.key === 'Backspace' && index > 0 && otp[index] === '') {
```

Comment:
Is there an enum for `'Backspace'` that we can use?

Reply:
Made an enum and used that instead of this